backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/MembershipImpl.java

```diff
@@ -206,7 +206,10 @@ public String getMemberElementId() {
      */
     @Override
     public String getMemberName() {
-        return memberName;
+        if (this.getMemberElement() != null) {
```

Comment:
Why do you modify this method? It is not derived, so I don't see any reason to modify it.